### DISCUSSION TITLE
Alien blood and "Bloody Business"

### DISCUSSION THREAD

**Discussion by Instant2116** [FakeFishGames/Barotrauma#13885] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Alien blood now deal organ damage instead of psychosis, but Medic (doctor) has talent "Bloody Business" that decrease psychosis from Alien blood, which doesn't make sense now.

### Reproduction steps

1. Start any type of game
2. Spawn medic
3. Read  "Bloody Business" talent description
4. It doesn't make sense

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Alien blood still gives psychosis as well, so it still makes sense.
```
      <StatusEffect tags="medical" type="OnSuccess" target="UseTarget, This" duration="15">
        <ReduceAffliction identifier="bloodloss" amount="6" />
        <Affliction identifier="psychosis" amount="3" />
        <Affliction identifier="organdamage" amount="0.65" />
      </StatusEffect>
      <StatusEffect tags="medical" type="OnFailure" target="UseTarget, This" duration="15">
        <ReduceAffliction identifier="bloodloss" amount="3" />
        <Affliction identifier="psychosis" amount="3" />
        <Affliction identifier="organdamage" amount="0.65" />
      </StatusEffect>
```

**Comment by JeroonBosch** [comment]
Closing (as my comment above explains, the talent still works to reduce psychosis and still makes sense)
